### ISSUE TITLE
✨ Develop the Breadcrumbs Component with Final Node Customization

### ISSUE THREAD

**Issue by bettybondoc** [SSWConsulting/SSW.Website#3326]
cc @wicksipedia @isaaclombardssw @UlyssesMaclaren @adamcogan @miichaelsmedley @louisroa8189 

**Description:**
Create the Breadcrumbs component for Consulting Pages to display a clear navigation hierarchy. The component should use the page title as the default final node, which should not be clickable. Customization options should allow users to edit the final node for SEO or usability purposes, with the ability to reset to the default.

**Acceptance Criteria**
- [x] The Breadcrumbs component includes the following default functionality:

>   - Displays the page hierarchy as breadcrumb links.
>   - The final node:
>     - Uses the page title by default.
>     - Is not clickable.
>   - Customization options:
>     - Edit the final node text (useful when the page title is too wordy for SEO or clarity).
>     - Reset the final node to its default value (page title).

- [x] Fully responsive across desktop, tablet, and mobile
- [x] Works seamlessly on all major browsers (Chrome, Firefox, Safari, Edge)
- [x] Fields and customization options can be enabled or disabled for flexibility
- [x] Code is ready for theming (e.g., Light/Dark modes, custom colors – see linked PBI: https://github.com/SSWConsulting/SSW.Website/issues/3300)
- [x] Tested and approved by Wicksy, Louis, and Betty

**Tasks**
- [x] Discuss with Wicksy and Betty best way to handle breadcrumbs (do we make it required for all pages?)
- [x] TODO - Developer to update

![Image](https://github.com/user-attachments/assets/61368d0a-3a3a-4962-ac65-f234bc67cb89)
**Figure: Breadcrumbs**

**Comment by isaaclombardssw** [comment]
This is pretty close to done, waiting for Caleb to do some of the harder parts as he would rather we go through it together (conditional collection displayed on /consulting).

Working on the simple parts of #3316 (ex. schema) in the meantime.

**Comment by Calinator444** [comment]
I've implemented the design and had it test passed by Betty and Louis, but unfortunately I needed to fix some bugs related to the traffic splitting for the new and old consulting pages so this won't be ready before for sprint review.

![Image](https://github.com/user-attachments/assets/2055012d-4c3b-495f-bca3-e4122e473239)

Also the linter is flagging "dark" as a non-tailwind class despite it being one https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually

![Image](https://github.com/user-attachments/assets/fcc05adc-1163-44b2-8c47-428963b4078f)

**Comment by Calinator444** [comment]
Also, I'll need to move the padding on the breadcrumbs into the component itself rather than the page layout. There are blocks that span the full width of the container that we need to accommodate for.

![Image](https://github.com/user-attachments/assets/24ae5da2-3920-42dd-932a-0674624b3bfe)

**Comment by Calinator444** [comment]
I haven't requested a test please yet because the playwright test for broken images was failing. The reason it's failing is because not all of the consulting pages are being generated with the new route splitting method. I found this out because I could see that next JS was failing to pre-fetch the data for the routes on the consulting index page. 

![Image](https://github.com/user-attachments/assets/6a1f77a0-dc70-463e-b068-04ac87ec7fef)


The reason the pages aren't being generated is that Tina's graphql client only returns the first 50 results for a particular query (unless overridden) and we have 94 consulting pages. I've fixed this issue and am currently unsure why the consulting pages still aren't being generated.

**Comment by Calinator444** [comment]
Done - You can use this component on the new consulting pages now

![Image](https://github.com/user-attachments/assets/3a0d57fe-c509-446c-a767-fdd1e5cb430e)
